### ISSUE TITLE
Console not working in Arch Linux

### ISSUE THREAD

**Issue by moul** [scaleway/image-archlinux#15]
There is not `getty` process running:

```console
[root@trusting-leakey ~]# ps auxwww | grep -i tty
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root      1278  0.0  0.0   4380   948 pts/0    S+   17:51   0:00 grep -i tty
```

**Comment by moul** [comment]
This can be fixed using `/sbin/agetty -L 9600 ttyS0 vt102 &`, now working to get this after each reboots

**Comment by moul** [comment]
You can fix it completely by using `systemctl enable serial-getty@ttyS0.service`

**Comment by moul** [comment]
```console
[root@lonely-turing ~]# systemctl disable serial-getty@.service
Removed symlink /etc/systemd/system/getty.target.wants/serial-getty@ttyS0.service.
[root@lonely-turing ~]# systemctl enable serial-getty@ttyS0.service
Created symlink from /etc/systemd/system/getty.target.wants/serial-getty@ttyS0.service to /etc/systemd/system/serial-getty@.service.
[root@lonely-turing ~]# systemctl restart serial-getty@ttyS0.service
```

@aimxhaisse can you check why your `systemctl enable serial-getty@ttyS0.service` in the Dockerfile seems to not be enough ?

**Comment by roock** [comment]
Same Issue here. I've solved it by changing the Linux Kenrel to 4.2 (latest).

**Comment by moul** [comment]
Thanks @roock, we will change the default bootscript to 4.2 for the next release

Just before this release, @aimxhaisse will check about this systemd strange behavior

**Comment by moul** [comment]
This is fixed using 226376e15eb6d138587fd4a6389debf107f18b37, the next build of the image will use a more recent kernel

For now you can change your bootscript manually: https://www.scaleway.com/docs/bootscript-and-how-to-use-it/
